alf/utils/value_ops.py

```diff
@@ -22,22 +22,21 @@
 
 
 def action_importance_ratio(action_distribution, collect_action_distribution,
-                            action, action_spec, calling_class_name, scope,
+                            action, action_spec, clipping_mode, scope,
                             importance_ratio_clipping, log_prob_clipping,
                             check_numerics, debug_summaries):
     """ ratio for importance sampling, used in PPO loss and vtrace loss.
 
-        Caller has to set tf.name_scope() and nest the call to this function
-            inside the with statement.
+        Caller has to save tf.name_scope() and pass scope to this function.
 
         Args:
             action_distribution: Probability of actions under target policy.
             collect_action_distribution: Probability of action from behavior
                 policy, used to sample actions for the rollout.
             action: action taken during rollout.
             action_spec (nested BoundedTensorSpec): representing the actions.
-            calling_class_name: to decide which clipping method to use.
-            scope: name_scope set outside.
+            clipping_mode: 1: double_sided [-x, +x] clipping for PPOLoss.
```

Comment:
Using a string instead of a number might have better readability. Also make the comment more specific:
mode for clipping the importance ratio. "double_sided" for clipping the importance ratio into range [1-x, 1+x], which is used by PPO algorithm.